src/services/WhitelistService.js

```diff
@@ -0,0 +1,15 @@
+import { feathersClient } from '../lib/feathersClient';
```

Comment:
Don't add it, we have whitelist provider already... use that.

Reply:
If you mean this

https://github.com/Giveth/giveth-dapp/blob/e8b4938644a07e99d8b0ed1063a41d9d8c29e3fc/src/contextProviders/WhiteListProvider.jsx#L19-L30


This function is not exported so I cant use that.

Reply:
WhitelistProvider export few functions, one is `isDelegate` which returns whether the user is a delegate or not